core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -3366,8 +3366,10 @@ class KafkaApisTest {
     when(clientQuotaManager.maybeRecordAndGetThrottleTimeMs(
       any[RequestChannel.Request](), anyDouble, anyLong)).thenReturn(0)
 
-    val fetchRequest = new FetchRequest.Builder(9, 9, -1, 100, 0, fetchDataBuilder)
+    val fetchRequest = new FetchRequest.Builder(9, 9, -1, -1, 100, 0, fetchDataBuilder)
       .build()
+    assertEquals(fetchRequest.replicaEpoch(), -1)
+    assertEquals(fetchRequest.replicaId(), -1)
```

Comment:
Those assertions are not really useful here because they don't test what the KafkaApis layer does. Let's remove them.